test/f90_correct/assign_cray_pointer_in_data.f

```diff
@@ -0,0 +1,7 @@
+       ! RUN: %flang -c %s -fsyntax-only
```

Comment:
It would be nice to move this test into `llvm_ir_correct` and actually check the LLVM IR that is expected from flang2.